integration_test/cloudbuild-integration-test.yaml

```diff
@@ -220,27 +227,32 @@
   - ${_STORE_NAME}
   - "--project=${_PROJECT}"
   - "--dataset=${_DATASET}"
-  - "--location=${_LOCATION}"  
+  - "--location=${_LOCATION}"
   - "--quiet"
   waitFor:
   - check-store-curl
   - run-find-scu
   - run-move-scu
   - run-commitment-scu
 
+- name: 'google/cloud-sdk:250.0.0'
```

Comment:
I think we want publishing images to be done post-submit rather than as a pre-submit. For that reason I think it's better to have the publish command as a separate .yaml file.

Reply:
not actually clear:
for the moment images published only if integration tests is successful
if we move publishing to separate file we'll have to build images again and run tests again it'll make new yaml almost exactly the same
or it can be manually triggered, or by tag, in this case we may remove file with version

Reply:
>Can we define a variable $PUBLISH

this will requre manipulation with trigger, not sure if it's very handy
What do you think about publishing by tag? if we have tag in commit we'll publish image after integration tests, also it's removes doubts about versions - tag contains clear string

Reply:
It would be very easy I think, since cloud build allows you to substitute variables for each trigger. We would simply set the publish variable to true for the post-submit run, and leave it as the default of false for the pre-submit.

If we publish in presubmits at all then we're publishing un-reviewed changes to the repo, right? Even if the commit ID is in the label instead of the version, users wouldn't be able to differentiate between a non-reviewed CL and a submitted CL.

Reply:
>It would be very easy I think

yeah, it's subjective

> we're publishing un-reviewed changes to the container repo, right?

yes

also i have idea how to simplify this process - we adopt git-flow for this repo and publish images only for updates in master branch, it means we'll publish only reviewed code
so for the moment we have 3 options

1. var in cloud build trigger
2. tag in commit  - same way as for https://github.com/GoogleCloudPlatform/wsi-to-dicom-converter/wiki/Release-process
3. changes in master - i think most clear way 

all of them quite simple to implement, please choose one

Reply:
I spoke to Dusan and he said we want to go with option #1, because the var in the cloud build trigger solves the issue of duplicating the YAML file, which I think we need to solve either way. This will also be essentially the same as option 3, as we are publishing a new container for each new commit to master - although we won't have separate dev and master branches.

Reply:
done